main.go

```diff
@@ -28,8 +29,20 @@ func main() {
 		log.Panicln(re)
 	}
 
+	ti, lu := os.LookupEnv("HTTP_TIMEOUT")
+	if lu {
+		log.Panicln(lu)
+	}
+
+	timeout, cv := strconv.Atoi(ti)
```

Comment:
Ce bloc devrait probablement être bougé dans le `if lu {}`. Si on a une variable d'environnement, parse it!